Frontend/src/pages/users.tsx

```diff
@@ -1,5 +1,5 @@
 import Head from "next/head";
-import NavBar from '../components/nav'
+import NavBar from '../components/Nav'
```

Comment:
these 2 parts

Reply:
the component is still showing up as nav.tsx in the code review editor

if theres no errors please ignore my comments 🦖